src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,15 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+
+        User user = UserService.findByEmail(email);
+        if (user != null) {
+
+            if (user.getPassword().equals(password)) {
+                return true;
+            }
+        }
+
```

Comment:
```suggestion
        User user = UserService.findByEmail(email);
        if (user != null) {
            if (user.getPassword().equals(password)) {
                return true;
            }
        }
```
remove redundant empty lines in all project
think how you can use & operator to simplify your code